sortingAlgorithms/quickSort/BLOG.md

```diff
@@ -0,0 +1,4 @@
+* I didn't complete this challenge yet.  I was having a hard time following the pseudocode.  I'm getting stuck at swap because I'm coming up with the same value for low and i, which means I'm just swapping the same value in the same place.
+
+
+![Whiteboard1](../../assets/quick-sort.jpg)
```

Comment:
This is a really big challenge, hopefully with code review today it will start to make more sense!
Some good websites to study:
- https://www.geeksforgeeks.org/quick-sort/
- Back to Back SWE on YouTube
- https://medium.com/@Charles_Stover/implementing-quicksort-in-javascript-8044a8e2bf39